src/vues/homologation.pug

```diff
@@ -24,7 +24,7 @@ block main
   .details-homologation.marges-fixes
     .tableau-bord
       h1!= homologation.nomService()
-      a.bouton(href = `/homologation/${homologation.id}/decision`) Voir le document d'homologation
+      a.bouton(href = `/homologation/${homologation.id}/decision`) Télécharger le dossier
```

Comment:
Ici on renvoi toujours vers la page _decision_ avec le récapitulatif affiché, est-ce le comportement souhaité ? Ou est-ce que l'on pourrait écrire `Voir le dossier` ?

Reply:
Sur le design cible, Mari a noté "Télécharger le dossier", après peut-être qu'elle avait en tête que le téléchargement se ferait directement depuis la synthèse du dossier. On peut laisser comme tel. Tu peux refuser ce commit uniquement ?

Reply:
oui on peu 👍🏼